include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -312,6 +325,8 @@ struct OrtApi {
 	*/
   OrtStatus*(ORT_API_CALL* AddCustomOpDomain)(_Inout_ OrtSessionOptions* options, _In_ OrtCustomOpDomain* custom_op_domain)NO_EXCEPTION;
 
+  OrtStatus*(ORT_API_CALL* RegisterCustomOpsLibrary)(_Inout_ OrtSessionOptions* options, _In_ const char* library_path)NO_EXCEPTION;
```

Comment:
Should we use ORTCHAR_T* here for consistency?

Reply:
We should, but Env::LoadDynamicLibrary only takes const char * vs the wchar_t * on Windows.

Reply:
Yeah, we should have both versions of LoadDynamicLibrary just like we've for FileOpenRd and FileOpenWr. If we change it to ORTCHAR_T* after 1.0 release, will it be ABI incompatible?